lib/internal/Magento/Framework/App/Test/Unit/HttpTest.php

```diff
@@ -155,6 +155,7 @@ private function setUpLaunch()
     public function testLaunchSuccess()
     {
         $this->setUpLaunch();
+        $this->requestMock->expects($this->once())->method('isHead')->will($this->returnValue(false));
```

Comment:
Please, use `willReturn` instead of `returnValue`

Reply:
Do you want me to change the rest of the test to use `willReturn` instead of `returnValue` also?

Reply:
It's better to refactor all such places but I don't insist and you can replace only in your changes.